en/docs/reference/synapse-properties/synapse-expressions.md

```diff
@@ -2,21 +2,42 @@
 
-Since JSONPath is limited to extracting JSON content only, it's necessary to use XPath to perform additional operations on the extracted data (for example, addition and substring extraction). Synapse Expressions address this limitation by enabling direct operations on JSON payloads, eliminating the need for complex workarounds.
+Since JSONPath is limited to extracting JSON content, it's often necessary to use XPath to perform additional operations on the extracted data (for example, addition and substring operations). Synapse Expressions address this limitation by enabling direct operations on JSON payloads, eliminating the need for complex workarounds.
 
 Furthermore, Synapse Expressions provide a unified syntax to access a wide range of system elements, including properties, variables, configurations, headers, registry content, secrets, and parameters. This cohesive approach simplifies development, making it significantly more efficient and user-friendly for developers.
 
-## Syntax
+The main building blocks of Synapse Expressions are expressions and parameters.
+
+<a href="{{base_path}}/assets/img/reference/expression_overview.png"><img src="{{base_path}}/assets/img/reference/expression_overview.png" alt="Expression Syntax" width="80%"></a>
+
+Each expression can just be a parameter or an expression which combines multiple parameters.
+
+We can chain the above building blocks to create complex expressions required by integration use cases. 
+
+Ex: The following expression is checking if the given customer is eligible for a promotion, based on number of orders he/she has placed and the total amount spent on a given order.
+
+```
+length(payload.orders[?(@.customerID == vars.customerId && @.total > params.queryParams.minimumBillAmount)]) > configs.promoEligibilityCount ? "Eligible" : "Not eligible";
+```
+
+<a href="{{base_path}}/assets/img/reference/complex_expression.png"><img src="{{base_path}}/assets/img/reference/complex_expression.png" alt="Expression Syntax" width="100%"></a>
+
+## Tooling support
 
-The syntax for Synapse Expressions is as follows:
+In places where the Synapse Expressions are used, the tooling support is provided to assist the user in writing the expressions. 
+
+<a href="{{base_path}}/assets/img/reference/VSCode_expression_support.gif"><img src="{{base_path}}/assets/img/reference/VSCode_expression_support.gif" alt="Expression Syntax" width="80%"></a>
+
+
+## Syntax
 
 ```
 ${expression}
 ```
 
-Here, the expression can be a combination of literals, value access expressions, operators, filter expressions, conditional expressions, and functions.
+Here, the expression can just be a parameter to access a value from the payload, properties, variables, headers, registry. Or it can be an expression which combines multiple parameters to perform operations on them.
```

Comment:
```suggestion
Here, the expression can just be a parameter to access a value from the payload, properties, variables, headers, or registry. Or, it can be an expression that combines multiple parameters to perform operations on them.
```